flink-cdc-runtime/src/main/java/com/ververica/cdc/runtime/operators/route/RouteFunction.java

```diff
@@ -96,6 +96,16 @@ public Event map(Event event) throws Exception {
             Selectors selectors = route.f0;
             TableId replaceBy = route.f1;
             if (selectors.isMatch(tableId)) {
+                // Add a rule that when configuring <> symbol in tablename,
+                // the namespace name and schemaName name needs to be changed
+                // the <> symbol needs to be replaced by the table name of the event
+                if (replaceBy.getTableName().contains("<>")) {
+                    replaceBy =
+                            TableId.parse(
+                                    replaceBy.getNamespace(),
+                                    replaceBy.getSchemaName(),
+                                    replaceBy.getTableName().replace("<>", tableId.getTableName()));
```

Comment:
We need to extract this literal into a static constant, and determine a value that will not be accidentally used in the table name.

Reply:
I changed a version and put the definition of the <> symbol into the configuration file, just like this form:
```
route:
  - source-table: mrtdb.[\S]*
    sink-table: mrtdb_sd.sd_s<>
    replace-symbol: s<>
```
In this way, the user can define the replacement symbols by themselves. Is it any better?

Reply:
+1. We just need to provide a default value.
Please add some tests for these scenes.

Reply:
Two methods,  RouteFunctionTest#testSchemaChangeRouting And RouteFunctionTest#testTableNameChangeRouting were modified to test this scenes.

Reply:
To expose this parameter to the users, please add description to [cdc-pipeline.md](https://github.com/ververica/flink-cdc-connectors/blob/15a734efa09343228dc7e69e2074a16bfec40f48/docs/content/overview/cdc-pipeline.md#route).
And add this configuration to [testParsingFullDefinition](https://github.com/ververica/flink-cdc-connectors/blob/15a734efa09343228dc7e69e2074a16bfec40f48/flink-cdc-cli/src/test/java/com/ververica/cdc/cli/parser/YamlPipelineDefinitionParserTest.java#L42).